tests/test_indexer.py

```diff
@@ -74,7 +73,12 @@ class TestIndexer(unittest.TestCase, DSSAsserts, StorageTestSupport):
 
     @classmethod
     def setUpClass(cls):
+        cls.replica = "aws"
```

Comment:
is this intended? if you're making this cloud-agnostic, shouldn't this test both code paths?

Reply:
That is intended for now. I'm working on making a base class for test_indexer to test any cloud service we use. That will be in my next PR.

Reply:
Cool, sgtm.